packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Guides/Installation" />
+
+# ⚙️ Installation
+<br/>
+<br/>
+
+## ✍🏻 Pre-requisite
+Before you install the package, make sure that you have performed following steps:
+Before using universe just ensure following things
+
+* You must be running Node version >=14.0.0
+* You must have `yarn` installed
+* Generate a Personal Access Token on GitHub by [visiting this link](https://github.com/settings/tokens/new?scopes=repo,workflow,write:packages,read:repo_hook,write:packages)
+  - From Enable SSO, click `Authorize` button next to Razorpay logo.
```

Comment:
This is a RZP specific thing and won't show up if someone from non RZP tries this.